c2rust-analyze/src/rewrite/expr/mod.rs

```diff
@@ -4,8 +4,10 @@ use rustc_hir::BodyId;
 use rustc_middle::mir::Body;
 use rustc_span::Span;
 
-mod hir_op;
```

Comment:
A bunch of other documentation (like the `mod rewrite` top-level docs) reference `hir_op` a lot, so could those be updated to reference the new `convert`, `distribute`, `unlower` `mod`s?

Reply:
I updated the `rewrite/mod.rs` doc comment, which is the only place I saw a mention of `hir_op`.

Reply:
I only actually saw it referenced there, so this should be perfect.

Reply:
Actually, could you also update 9d2510327274776803bacfb0248870cf67c574eb to use intra-doc links (e.x. `` [`item`] `` style)?  That way such dangling references would be caught automatically.

Reply:
Rustdoc doesn't seem to like those links, presumably because the targets are private - it reports "no item named `convert` in module `expr`" etc.

Reply:
Hmm, that's annoying, oh well.